packages/components/src/notice/index.js

```diff
@@ -84,6 +88,16 @@ function Notice( {
 		children = <RawHTML>{ children }</RawHTML>;
 	}
 
+	onDismiss = onDismiss || noop;
```

Comment:
After removing the default null value, I ran the tests for notices as well as snackbar and it works fine. 
The null value was doing nothing since onDismiss is also handled independently in snackbar and now in notices. 🙂

Reply:
Yes. You'll have to update the failing tests though that expect `onDismiss === null` to `undefined`.

Reply:
I just updated the tests. Sorry for the late reply.